src/Symfony/Component/Intl/Tests/NumberFormatter/AbstractNumberFormatterTest.php

```diff
@@ -428,6 +428,7 @@ public function formatRoundingModeRoundHalfUpProvider()
             // array(1.125, '1.13'),
             array(1.127, '1.13'),
             array(1.129, '1.13'),
+            array(1020 / 100, '10.20'),
```

Comment:
why not `10.2` rather than a division ?

Reply:
@stof that's the issue, `1020 / 100 != 10.2`, the actual value is `10.19999999998`

You can test this example:
```
echo floor(1020 / 100 * 100); // 1019
echo floor((float) (string) (1020 / 100 * 100)); // 1020
```

See the original PR with this sort of fix #24036.